File: src/main/java/com/dumbdogdiner/warrior/commands/warrior/WarriorHologramCommand.java
package com.dumbdogdiner.warrior.commands.warrior;

import com.dumbdogdiner.warrior.api.command.SubCommand;
import org.bukkit.command.CommandSender;

import java.util.ArrayList;
import java.util.List;

public class WarriorHologramCommand implements SubCommand {

    @Override
    public String getAlias() {
        return "hologram";
    }

    @Override
    public String getSyntax() {
        return "/warrior hologram <stats|leaderboard>";
    }

    @Override
    public String getPermission() {
        return "warrior.command.hologram";
    }

    @Override
    public boolean execute(CommandSender sender, String commandLabel, String[] args) {
        return true;
    }

    @Override
    public List<String> getArguments(CommandSender sender, String[] arguments) {
        return new ArrayList<>();
    }

}

File: warrior/src/main/kotlin/com/dumbdogdiner/warrior/commands/warrior/WarriorHologramCommand.kt
package com.dumbdogdiner.warrior.commands.warrior

import com.dumbdogdiner.warrior.api.command.SubCommand
import org.bukkit.command.CommandSender
import java.util.ArrayList

class WarriorHologramCommand : SubCommand {
    override fun getAlias(): String {
        return "hologram"
    }

    override fun getSyntax(): String {
        return "/warrior hologram <stats|leaderboard>"
    }

    override fun getPermission(): String {
        return "warrior.command.hologram"
    }

    override fun execute(sender: CommandSender, commandLabel: String, args: Array<String>): Boolean {
        // TODO: implement hologram spawner
        return true
    }

    override fun getArguments(sender: CommandSender, arguments: Array<String>): List<String> {
        return ArrayList()
    }
}